specifyweb/frontend/js_src/lib/components/WorkBench/DataSetMeta.tsx

```diff
@@ -36,354 +36,420 @@ import { LocalizedString } from 'typesafe-i18n';
-  );
+              > =>
+                fetchCollection('SpecifyUser', {limit: 500 }).then(({records: users }) =>
+                users.filter(({id}) => id !== userInformation.id)
+                );
 
-function ChangeOwner({
-  dataset,
-  onClose: handleClose,
+                function ChangeOwner({
+                  dataset,
+                  onClose: handleClose,
 }: {
-  readonly dataset: Dataset;
+                  readonly dataset: Dataset;
   readonly onClose: () => void;
 }): JSX.Element | null {
   const [users] = useAsyncState<RA<SerializedResource<SpecifyUser>>>(
-    fetchListOfUsers,
-    true
-  );
+                  fetchListOfUsers,
+                  true
+                  );
 
-  const id = useId('change-data-set-owner');
-  const [newOwner, setNewOwner] = React.useState<number | undefined>(undefined);
-  const [isChanged, setIsChanged] = React.useState(false);
-  const loading = React.useContext(LoadingContext);
+                  const id = useId('change-data-set-owner');
+                  const [newOwner, setNewOwner] = React.useState<number | undefined>(undefined);
+                  const [isChanged, setIsChanged] = React.useState(false);
+                  const loading = React.useContext(LoadingContext);
 
-  return users === undefined ? null : isChanged ? (
-    <Dialog
-      buttons={commonText.close()}
-      header={wbText.dataSetOwnerChanged()}
-      onClose={(): void => unsafeNavigate('/specify/', { replace: true })}
-    >
-      <p>{wbText.dataSetOwnerChanged()}</p>
-    </Dialog>
-  ) : (
-    <Dialog
-      buttons={
-        <>
-          <Button.DialogClose>{commonText.cancel()}</Button.DialogClose>
-          <Submit.Blue disabled={newOwner === undefined} form={id('form')}>
-            {wbText.changeOwner()}
-          </Submit.Blue>
-        </>
-      }
-      header={wbText.changeDataSetOwner()}
-      onClose={handleClose}
-    >
-      <Form
-        id={id('form')}
-        onSubmit={(): void =>
-          loading(
-            ping(
-              `/api/workbench/transfer/${dataset.id}/`,
-              {
-                method: 'POST',
-                body: formData({
-                  specifyuserid: newOwner!,
-                }),
-              },
-              { expectedResponseCodes: [Http.NO_CONTENT] }
-            ).then(() => setIsChanged(true))
-          )
-        }
-      >
-        <Label.Block>
-          <p>{wbText.changeDataSetOwnerDescription()}</p>
-          <Select
-            size={10}
-            value={newOwner}
-            onChange={({ target }): void =>
-              setNewOwner(Number.parseInt(target.value))
-            }
-          >
-            {users.map((user) => (
-              <option key={user.id} value={user.id}>
-                {user.name}
-              </option>
-            ))}
-          </Select>
-        </Label.Block>
-      </Form>
-    </Dialog>
-  );
+                  return users === undefined ? null : isChanged ? (
```

Comment:
Looks like the code was reformatted inconsistently.
Could you configure the Prettier plugin (https://github.com/specify/specify7/tree/production/specifyweb/frontend/js_src#plugins) in your IDE and reformat this file with it?